### PR TITLE
Feature/robot  giftentry automation

### PR DESCRIPTION
**Pull request by skristem**
Added a new test case for Gift Entry Feature
1. Verify Error message is displayed when GE is disabled
2. Enable GE and verify a default template is visible on the page
# Critical Changes

# Changes

# Issues Closed

# Community Ideas Delivered

# New Metadata

# Deleted Metadata

### CODE REVIEW

File: robot/Cumulus/tests/browser/gift_entry/1_enable_ge.robot [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+...             robot/Cumulus/resources/GiftEntryPageObject.py
+Suite Setup     Open Test Browser
+Suite Teardown  Capture Screenshot and Delete Records and Close Browser
+
+*** Test Cases ***
+
+Verify Error Message with GE Disabled and enable GE
+    [Documentation]                           This test case checks if advanced mapping is enabled and if disabled, enables it. 
+    ...                                       Checks if gift entry is disabled and if enabled, disables it and verifies that GE page has error. 
+    ...                                       Then enables gift entry and veries that gift entry page lands on templates page and has default template
```
Comment (sgovindankutty) [inline robot/Cumulus/tests/browser/gift_entry/1_enable_ge.robot:15]: Small correction! `verifies` is misspelled :)

File: robot/Cumulus/tests/browser/gift_entry/1_enable_ge.robot [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    Go To Page                                Custom              GE_Gift_Entry
+    Wait Until Page Contains                  This feature requires both Advanced Mapping and Gift Entry. Please enable them in NPSP Settings.
+    Go To Page                                Custom              NPSP_Settings
+    Open Main Menu                            System Tools
+    Click Link With Text                      Advanced Mapping for Data Import & Gift Entry
+    Click Toggle Button                       Gift Entry
+    Wait For Message                          Gift Entry Enabled                              
+    Go To Page                                Custom              GE_Gift_Entry
+    Wait Until Page Contains                  Templates
+    Wait Until Page Contains                  Default Gift Entry Template
```
Comment (sgovindankutty) [inline robot/Cumulus/tests/browser/gift_entry/1_enable_ge.robot:31]: Are these two different pages ?

Comment (skristem) [reply robot/Cumulus/tests/browser/gift_entry/1_enable_ge.robot:31]: Its the same page, Just making sure its landing on Templates page and displaying the Default Gift Entry Template

File: robot/Cumulus/tests/browser/gift_entry/1_enable_ge.robot (hunk 1 above)
Comment (skristem) [reply robot/Cumulus/tests/browser/gift_entry/1_enable_ge.robot:15]: thanks will correct it

**Pull request merged**